skyvern/library/skyvern.py

```
@@ -95,127 +93,125 @@ def __init__(
-                timeout=timeout,
-                follow_redirects=follow_redirects,
-                httpx_client=create_embedded_server(
-                    llm_config=llm_config,
-                    settings_overrides=settings,
-                ),
-            )
-        else:
-            if not api_key:
-                raise ValueError(f"Missing api_key for {environment.name}")
+        if not os.path.exists(".env"):
+            raise ValueError("Please run `skyvern quickstart` to set up your local Skyvern environment")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)